src/kagglehub/handle.py

```diff
@@ -72,6 +70,20 @@ def to_url(self) -> str:
         return base_url
 
 
+@dataclass
+class CompetitionHandle(ResourceHandle):
+    competition: str
+
+    def __str__(self) -> str:
+        handle_str = f"competitions/{self.competition}"
```

Comment:
We don't include the prefix in the handle: https://github.com/Kaggle/kagglehub/blob/7028ed214b815b05d29521d02082489690952a94/src/kagglehub/handle.py#L62

Reply:
d.